contributing-device-support.md

```diff
@@ -0,0 +1,125 @@
+# Contributing new board support to resin.io
+
+Pre-requisites: yocto BSP layer for your particular board, supplying the BSP (board support package). Should be compatible with yocto daisy/fido/jethro releases.
```

Comment:
We should state the yocto versions generic. Something like "compatible to the yocto releases resin currently supports".